### PR TITLE
DrawerLayoutAndroid

### PR DESCRIPTION
**Pull request by arian**
DrawerLayoutAndroid has a static property (`positions`), that is mocked now as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "prepublish": "npm run build",
     "test": "npm run lint && npm run mocha",
-    "mocha": "mocha --require babel-core/register test/**/*.js",
+    "mocha": "mocha --require babel-core/register ./test/**/*.js ./test/*.js",
```
Comment (RealOrangeOne) [inline package.json]: i'm not sure this change is necessary, both my machine and Travis have no problem running the tests in their current setup

Comment (arian) [reply package.json]: true, for some reason it only ran 2 or 4 tests for me... I'll re-check.

Comment (arian) [reply package.json]: Seems the glob pattern should be between single quotes: http://stackoverflow.com/a/34529436

Comment (RealOrangeOne) [reply package.json]: oh, should it. I never realised that. Perfect!

File: src/NativeModules/UIManager.js [hunk 2]
```diff
@@ -61,6 +61,11 @@ const UIManager = {
   configureNextLayoutAnimation(callback, errorCallback) {
 
   },
+  AndroidDrawerLayout: {
+    Constants: {
+      DrawerPosition: { Left: 8388611, Right: 8388613 },
```
Comment (RealOrangeOne) [inline src/NativeModules/UIManager.js:66]: Where are you getting these numbers from. Not sure I like the use of _magic numbers_.

Comment (arian) [reply src/NativeModules/UIManager.js:66]: I know: https://developer.android.com/reference/android/view/Gravity.html#END and https://developer.android.com/reference/android/view/Gravity.html#START

That are the values used in the AndroidDrawerLayout: https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/views/drawer/ReactDrawerLayoutManager.java#L145

Comment (arian) [reply src/NativeModules/UIManager.js:66]: I think for mocking, the important thing is that they exist right?

Comment (RealOrangeOne) [reply src/NativeModules/UIManager.js:66]: Yes, but it would be better if they were accurate. Well, if it's the values they use, it's good enough for me!

**Pull request merged**